webtestingii/src/App.test.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { render, fireEvent } from "react-testing-library";
+import "jest-dom/extend-expect";
+import App from "./App";
+
+describe("<App />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div");
+    ReactDOM.render(<App />, div);
+    ReactDOM.unmountComponentAtNode(div);
+  });
+
+  it("renders successfully", () => {
+    render(<App />);
+  });
+
+  it("successfully strikes", () => {
+    const { getAllByText, getByText } = render(<App />);
+    const button = getAllByText(/strike/i)[1];
```

Comment:
ah, so this will grab the second node that matches that regex string, aka the button? nicely done. on one hand it's kind of "brittle" testing as Luis likes to say since if anybody adds any other feature that says the word strikes this will probably break, but this is fine in the app as it is